ArcGISRuntimeSDKQt_CppSamples/Scenes/ScenePropertiesExpressions/README.md

```diff
@@ -0,0 +1,20 @@
+# Scene Properties Expressions
+
+This sample demonstrates how to update the orientation of a graphic using scene property rotation expressions via QSlider.
```

Comment:
I don't think the type is called `QSlider` for qt quick. Also, the point of the sample is not that we use a slider - just that we update the properties, so how about just:

> This sample demonstrates how to update the orientation of a graphic using scene property rotation expressions.

Reply:
Done.